README.md

```diff
@@ -5,11 +5,11 @@
 [![Issue Stats](http://issuestats.com/github/theforeman/foreman/badge/pr)](http://issuestats.com/github/theforeman/foreman)
 [![Support IRC channel](https://kiwiirc.com/buttons/irc.freenode.net/theforeman.png)](https://kiwiirc.com/client/irc.freenode.net/?#theforeman)
 
-[Foreman](http://theforeman.org) is a free open source project that gives you the power to easily **automate repetitive tasks**, quickly **deploy applications**, and proactively **manage your servers lifecyle**, on-premises or in the cloud.
+[Foreman](https://theforeman.org) is a free open source project that gives you the power to easily **automate repetitive tasks**, quickly **deploy applications**, and proactively **manage your servers lifecyle**, on-premises or in the cloud.
 
 From **provisioning** and **configuration** to **orchestration** and **monitoring**, Foreman integrates with your existing infrastructure to make operations easier.
 
-* Website: [theforeman.org](http://theforeman.org)
+* Website: [theforeman.org](https://theforeman.org)
 * ServerFault tag: [Foreman](http://serverfault.com/questions/tagged/foreman)
 * Issues: [Redmine](http://projects.theforeman.org/issues)
 * Wiki: [Foreman wiki](http://projects.theforeman.org/projects/foreman/wiki/About)
```

Comment:
The freenode link (line below) below doesn't work. I know you didn't break it, but while we're at it...